src/main/java/io/github/skyblockcore/util/APIUtils.java

```diff
@@ -0,0 +1,71 @@
+package io.github.skyblockcore.util;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class APIUtils {
+
+    private static JsonElement PlayerInfo;
+    private static String Player;
+
+    private static HashMap<String, String> PlayerData = new HashMap<String, String>();
```

Comment:
What's the naming convention with these variables doing?

Reply:
Will fix but any suggestions I can't think of anything

Reply:
> Will fix but any suggestions I can't think of anything

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html